### PR TITLE
feat(docs-infra): update to @angular/* v7

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -94,10 +94,10 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.9.0-rc.1",
```
Comment (gkalpak) [inline aio/package.json]: Why not this one too? :grin:

**Review by gkalpak — APPROVED** [review]

**Comment by IgorMinar** [comment]
caretaker note: ci flake

File: aio/package.json (hunk 1 above)
Comment (IgorMinar) [reply aio/package.json]: uh... good question. is this related to https://github.com/angular/angular-cli/issues/12624 ?

File: aio/package.json [hunk 2]
```diff
@@ -93,7 +93,7 @@
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.9.0-rc.1",
+    "@angular-devkit/build-angular": "~0.10.0",
```
Comment (gkalpak) [inline aio/package.json]: :+1:

**Pull request closed without merging**